package_manifest.yaml

```diff
@@ -310,6 +174,187 @@ foreman_packages:
+    nodejs-babel-preset-env: {}
+    nodejs-babel-preset-react: {}
+    nodejs-babel-register: {}
+    nodejs-babel-runtime: {}
+    nodejs-bootstrap-slider-without-jquery: {}
+    nodejs-brace: {}
+    nodejs-breakjs: {}
+    nodejs-c3: {}
+    nodejs-change-emitter: {}
+    nodejs-classnames: {}
+    nodejs-compression-webpack-plugin: {}
+    nodejs-core-js: {}
+    nodejs-css-element-queries: {}
+    nodejs-css-loader: {}
+    nodejs-d3: {}
+    nodejs-datatables.net-bs: {}
+    nodejs-datatables.net: {}
+    nodejs-debug: {}
+    nodejs-diff: {}
+    nodejs-dotenv: {}
+    nodejs-expose-loader: {}
+    nodejs-extract-text-webpack-plugin: {}
+    nodejs-fbjs: {}
+    nodejs-file-loader: {}
+    nodejs-follow-redirects: {}
+    nodejs-harmony-reflect: {}
+    nodejs-hoist-non-react-statics: {}
+    nodejs-identity-obj-proxy: {}
+    nodejs-ipaddr.js: {}
+    nodejs-is-buffer: {}
+    nodejs-isomorphic-fetch: {}
+    nodejs-jquery-flot: {}
+    nodejs-jquery-ujs: {}
+    nodejs-jquery.cookie: {}
+    nodejs-jquery: {}
+    nodejs-js-tokens: {}
+    nodejs-jsdom: {}
+    nodejs-jstz: {}
+    nodejs-lodash-es: {}
+    nodejs-lodash-webpack-plugin: {}
+    nodejs-lodash.debounce: {}
+    nodejs-lodash.orderby: {}
+    nodejs-lodash: {}
+    nodejs-loose-envify: {}
+    nodejs-ms: {}
+    nodejs-multiselect: {}
+    nodejs-node-gyp: {}
+    nodejs-node-sass: {}
+    nodejs-object-assign: {}
+    nodejs-patternfly-react: {}
+    nodejs-patternfly: {}
+    nodejs-performance-now: {}
+    nodejs-prop-types: {}
+    nodejs-raf: {}
+    nodejs-react-bootstrap-switch: {}
+    nodejs-react-bootstrap: {}
+    nodejs-react-c3js: {}
+    nodejs-react-debounce-input: {}
+    nodejs-react-dom: {}
+    nodejs-react-ellipsis-with-tooltip: {}
+    nodejs-react-fontawesome: {}
+    nodejs-react-numeric-input: {}
+    nodejs-react-onclickoutside: {}
+    nodejs-react-password-strength: {}
+    nodejs-react-redux: {}
+    nodejs-react: {}
+    nodejs-reactabular-table: {}
+    nodejs-recompose: {}
+    nodejs-redux-form-validators: {}
+    nodejs-redux-form: {}
+    nodejs-redux-logger: {}
+    nodejs-redux-thunk: {}
+    nodejs-redux: {}
+    nodejs-regenerator-runtime: {}
+    nodejs-sass-loader: {}
+    nodejs-seamless-immutable: {}
+    nodejs-select2: {}
+    nodejs-sortabular: {}
+    nodejs-style-loader: {}
+    nodejs-symbol-observable: {}
+    nodejs-table-resolver: {}
+    nodejs-uglifyjs-webpack-plugin: {}
+    nodejs-urijs: {}
+    nodejs-url-loader: {}
+    nodejs-uuid: {}
+    nodejs-webpack-stats-plugin: {}
+    nodejs-webpack: {}
+    nodejs-zxcvbn: {}
+    pcp-mmvstatsd: {}
+    puppet-agent-oauth: {}
+    puppet-agent-puppet-strings: {}
+    puppet-agent-rgen: {}
+    puppet-agent-yard: {}
+    rubygem-concurrent-ruby: {}
+    rubygem-foreman_maintain: {}
+    rubygem-kafo: {}
+    rubygem-kafo_parsers: {}
+    rubygem-kafo_wizards: {}
+    rubygem-rb-inotify: {}
+    rubygem-rsec: {}
+    rubygem-rubyipmi: {}
+
+foreman_scl_nonscl_packages:
```

Comment:
Is there anyway to rename this? At a first glance it feels like the name is negating itself. Is it supposed to be packages in the foreman scl that are not strictly needed as apart of the scl?

Reply:
Would `_and_` help? Other suggestions?

Reply:
`foreman_scl_and_nonscl_packages` or `foreman_mixed_scl_nonscl_packages` works for me too.